calico/felix/fiptables.py

```diff
@@ -57,25 +59,26 @@ def on_endpoint_chains_ready(self, iface_name, endpoint_id):
         :param iface_name: name of the linux interface.
         :param endpoint_id: ID of the endpoint, used to form the chain names.
         """
-        if self.iface_to_ep_id_suffix.get(iface_name) != endpoint_id:
-            self.iface_to_ep_id_suffix[iface_name] = endpoint_id
+        if self.iface_to_ep_id.get(iface_name) != endpoint_id:
```

Comment:
I think this hasn't been renamed in the __init__.  If the chain names are going to be based on the iface name only then maybe we just need a set of those rather than a map to ep id?